leku/src/main/res/layout-land/activity_location_picker.xml

```diff
@@ -112,6 +113,37 @@
             >
 
           <TextView
```

Comment:
Instead of adding the views directly to the layout maybe could be better extract them to a new layout and include the into the view.